src/morphs.jl

```diff
@@ -42,6 +42,7 @@ function morph_to(to_func::Function; do_action = :stroke)
         _morph_to(video, object, action, frame, to_func; do_action = do_action)
 end
 
+const PREVIOUS_MORPH_ACTION = Array{Action,1}()
```

Comment:
I assume this isn't working when you more several objects at the same time.
I think this should be part of `opts` so per object